example/cumulus-tf/variables.tf

```diff
@@ -24,6 +24,25 @@ variable "cumulus_message_adapter_lambda_layer_arn" {
   type = string
 }
 
+variable "cmr_oauth_provider" {
+  type = string
+  default = "earthdata"
+}
+
+variable "launchpad_api" {
+  type = string
+  default = "launchpadApi"
+}
+
+variable "launchpad_certificate" {
+  type = string
+  default = "launchpad.pfx"
+}
+
+variable "launchpad_passphrase" {
+  type = string
```

Comment:
Can this have a default value of an empty string? Otherwise it's required even if you don't want to deploy with launchpad integration